Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp

```diff
@@ -1907,6 +1907,7 @@ WebsiteDataStoreParameters WebsiteDataStore::parameters()
 #if !HAVE(NSURLSESSION_WEBSOCKET)
     networkSessionParameters.shouldAcceptInsecureCertificatesForWebSockets = m_configuration->shouldAcceptInsecureCertificatesForWebSockets();
 #endif
+    networkSessionParameters.blobRegistryTopOriginPartitioningEnabled = WebProcessProxy::isBlobRegistryPartitioningEnabled();
```

Comment:
That's not quite right, we only want to iterate over the pages that are associated with this data store, not ALL the pages.

Reply:
Right, good point, and in the other case (when a webpage is added/removed), do we want to look at all pages or only pages in that same web process or all pages associated with the same datastore? It seems like we want similar behavior in both places. I believe in both cases we want to look at all pages that are associated with the same data store. Does that make sense and do you agree?

Reply:
Sorry, I didn't see your last review until after I pushed a new version and asked this question. Your review comments were helpful, thanks.